index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Poetry</title>
+  </head>
+   <body style="background-color:lightblue;">
+  <body>
+    
+    <p>
+      <h1> 
+  <strong>ON PAPER</strong>
+    </h1> 
+    </p>
+    
+<p>
+ <h2>
+   By Jacqueline Woodson
+  </h2> 
+    </p>
+
+ <p>
+   The first time I write my full name
+    </p>
+<p>
+  Jacqueline Amanda Woodson
+    </p>
+ <p>
+   without anybody's help
+    </p>
+ <p>
+   on a clean white page in my composition notebook,
+    </p>
+ <p>
+   I know  
+     </p>
+<p>
+   if I wanted to
+</p>
+<p>
+  I could write anything.
+    </p>
+<p>
+  Letters becoming words, words gathering meaning,
+    </p>
+<p>
+  becoming thoughts outside my head
+    </p>
+<p> <!--  🧠 🗣 -->
+    </p>  
+<p>
+  becoming sentences
+    </p>
+<p>
+  <p class="written-by">
+  <strong>Written by</strong>
+    </p>
+
+<p>
+  <p class="jac">
+  Jacqueline Amanda Woodson
+</p>  
+
+
+<img style="float: right;" alt="src=https://rockysmith.files.wordpress.com/2020/07/woodson-j.jpg>
+
+
+<div style="clear: right">
+
+</div>    
```

Comment:
This is a less-than-ideal way to clear your floated element. Instead, you should use the [clearfix styles](https://mica-web.github.io/learn/css/best-practices#wrap-all-floated-elements-in-another-element-with-the-clearfix-class-applied) in the CSS best practices.